web_action_conditionable/static/description/index.html~2eb1f1bc... [FIX] web_action_conditionable: Adapt to new api

```diff
@@ -0,0 +1,435 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
I doubt this file is needed

Reply:
I am not sure, maybe @Tardo can shed some light here

Reply:
This looks like a temporary file... :fire:

Reply:
thank you